ui/pages/confirmations/components/confirm/info/approve/approve.test.tsx

```diff
@@ -56,7 +55,10 @@ describe('<ApproveInfo />', () => {
       },
     };
```

Comment:
Hey @pedronfigueiredo : `waitFor` above is resulting in 4 snapshots being written. Is that intended ?